Makefile

```diff
@@ -65,18 +65,24 @@ ifeq ($(PULL_ARTIFACTS),no)
 endif
 
 build:
-ifeq ($(PULL_ARTIFACTS),yes)
-	python3 -u ./py/confluent/build/download_artifacts/download_artifacts.py
-endif
-
 ifeq ($(SKIP_TESTS),yes)
 	mvn -DskipTests=true install
 else
 	mvn install
 endif
 
-install: build
+newbuild:
+	git clone git@github.com:confluentinc/download-artifacts.git
```

Comment:
1. do these need their own dedicated repo or could we just put them, e.g., release-tools
2. do we need versioning on these to maintain reproducible  builds? for example, what if on a future release branch we need to  change behavior of the download scripts, but we still need to be able to release older branches. The downside however, is if these versions get scattered through all the repos, it's a pain to update. (An alternative if we do keep the scripts in  their own repo is to manage versions along with product version  branches and tags, i.e. 5.1.x, 5.2.x, etc, and branch them with tbr like we do for the product repos).

Reply:
One potential solution I am thinking is that keep my script independent from release-tools and use same version branch as CP to control behavior.